framework/include/utils/InputParameters.h

```diff
@@ -1268,7 +1315,10 @@ template <typename T>
 const T &
 InputParameters::getParamHelper(const std::string & name, const InputParameters & pars, const T *)
 {
-  if (!pars.isParamValid(name))
+  // TODO: after updating apps (i.e. rattlesnake) remove the first if clause+body:
+  if (name == "parser_syntax")
+    return (const T &)pars.blockFullpath();
```

Comment:
Note: We normally don't use C-style casts anywhere in MOOSE. Since this line of code is temporary, I'll let it slide. In the future, use the (longer) C++-style casts.